### ISSUE TITLE
Insufficient fix for #1126

### ISSUE THREAD

**Issue by karelorigin** [traefik/yaegi#1350]
Insufficient fix for #1126

### The following program `sample.go` triggers an unexpected result

```go
package main

import (
	"errors"
	"fmt"
	"strings"
)

func main() {
	err := errors.New("hello there")

	switch true {
	case strings.Contains(err.Error(), "hello"):
		fmt.Println("True!")
	default:
		fmt.Println("False!")
	}
}
```

### Expected result

```console
$ go run main.go
// True!
```

### Got

```console
$ yaegi main.go
// False!
```

### Yaegi Version

v0.11.1

### Additional Notes

Almost identical to #1126, except that the first condition is always ignored. A temporary fix would be to add a placeholder:

```golang
	switch true {
        case false:
	case strings.Contains(err.Error(), "hello"):
		fmt.Println("True!")
	default:
		fmt.Println("False!")
	}
```

**Comment by karelorigin** [comment]
Any news on this? It's a core issue, definitely seems important

**Comment by mvertes** [comment]
This is fixed by the PR #1370, part of v0.11.3
